.github/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### New features
 
+* Fixes bug in `strawberryfields.decompositions.rectangular_symmetric` so its
+  returned phases are all in the interval [0, 2*pi[, and corrects the
+  function docstring.
```

Comment:
```suggestion
  function docstring.
  [#196](https://github.com/XanaduAI/strawberryfields/pull/196)
```

Reply:
Also, could you please move this down into the 'bug fixes' part of the changelog. Thanks!